### PR TITLE
use expression based properties

### PR DESCRIPTION
**Pull request by SimonCropp**

### CODE REVIEW

File: Source/Csla.test/MethodCaller/PropertyTests.cs [hunk 1]
```diff
@@ -16,11 +16,13 @@ public class PropertyTests
     private string one;
     private string three;
 
-    public string Test1 { get { return "one"; } }
+    public string Test1 => "one";
 
-    public string Test2 { get { return "two"; } set { one = value; } }
+    public string Test2 { get => "two";
```
Comment (StefanOssendorf) [inline Source/Csla.test/MethodCaller/PropertyTests.cs:21]: Put get on it's own line.

File: Source/Csla.test/ValidationRules/ValidationTests.cs [hunk 2]
```diff
@@ -768,14 +768,11 @@ public class HasLazyField : BusinessBase<HasLazyField>
...
-      get
-      {
-        return LazyGetProperty(Value1Property, () => string.Empty);
-        //if (!FieldManager.FieldExists(Value1Property))
-        //  SetProperty(Value1Property, string.Empty);
-        //return GetProperty(Value1Property);
-      }
-      set { SetProperty(Value1Property, value); }
+      get => LazyGetProperty(Value1Property, () => string.Empty);
+      //if (!FieldManager.FieldExists(Value1Property))
```
Comment (StefanOssendorf) [inline Source/Csla.test/ValidationRules/ValidationTests.cs:772]: This is now confusing with the change. This belongs to the `get` but reads without knowledge as it belongs to the `set`. So just delete it.

**Review by StefanOssendorf — CHANGES_REQUESTED** [review]
I'm in general okay with this change. Last decision lies by @rockfordlhotka 

@SimonCropp Could you also please add this change to the editorconfig so it'll b suggested by VS automatically? Thanks

**Comment by SimonCropp** [comment]
> Could you also please add this change to the editorconfig so it'll b suggested by VS automatically

done

**Review by rockfordlhotka — APPROVED** [review]

**Comment by rockfordlhotka** [comment]
@StefanOssendorf I'm good with this change - I like the language feature. Once your change request is satisfied it can be merged.

**Comment by StefanOssendorf** [comment]
@SimonCropp Please fix the conflicts

**Comment by rockfordlhotka** [comment]
Hi @SimonCropp 

Do you plan to continue working on this PR - to resolve the conflicts - or should I close it without merging?

**Comment by SimonCropp** [comment]
with that number of conflicts, it is prob easier to close this and resubmit

**Pull request closed without merging**